lego/apps/surveys/fixtures/survey_template_fixtures.yaml

```diff
@@ -0,0 +1,240 @@
+- model: surveys.Survey
+  pk: 1
+  fields:
+    title: 'Template: company_presentation'
+    event: 1
+    template_type: 'company_presentation'
```

Comment:
I intend to load these template fixtures in prod, ala `./manage.py loaddata lego/apps/surveys/fixtures/survey_template_fixtures.yaml`. That's fine, right?